doc/api_ref/ffi.rst

```diff
@@ -249,6 +249,7 @@ supported it.
 ============== ===================
 FFI Version    Supported Starting
 ============== ===================
+20260203       3.11.0
```

Comment:
I personally much prefer using the "intended" releases dates, since when searching for a version you can just look at your calendar instead of having to go to the repo, but if you'd rather have "today", let me know.